tests/infra/ccf.py

```diff
@@ -333,6 +307,32 @@ def create_users(self, users):
 
     # TODO: The following governance functions should be moved to their own class
     # See https://github.com/microsoft/CCF/issues/364
+    def check_for_service(self, node, status=ServiceStatus.OPEN):
```

Comment:
This function already existed. I've just moved it here as it is a governance function.